tools/local-dev.sh

```diff
@@ -1,9 +1,24 @@
 #!/bin/bash
+set -e
```

Comment:
Since this is being sourced, this will set errexit in the shell, even after the script exits. I think this would be an unintended side-effect.

Reply:
Thanks, I totally missed this and it was causing me problems. I've removed the `set -e`.